packages/@tinacms/cli/package.json

```diff
@@ -78,7 +78,7 @@
 		"cors": "^2.8.5",
 		"crypto-js": "^4.2.0",
 		"dotenv": "^16.4.5",
-		"esbuild": "^0.18.20",
```

Comment:
**question**: Should the version of `esbuild` included here match the 1 in the [[at]tinacms/scripts](https://github.com/tinacms/tinacms/blob/67b87d8bbcdc6ff660194869e72d2a204e0d5d93/packages/%40tinacms/scripts/package.json#L25) package?

Reply:
Done update the scripts packages to use the same version of esbuild